### DISCUSSION TITLE
Custom Tables and Relationships

### DISCUSSION THREAD

**Discussion by jcampbell05** [pods-framework/pods#6644] [Ideas]
Is there a way to make my own Table work with Pods like a way to export them into a format I like? or something ? 

It would be nice if the relationships between Custom Tables could be in a custom table rather than in the standard Wordpress table.

Would I need to roll it out myself ?

**Comment by sc0ttkclark** [comment]
Trying to understand what you're looking for here better. Are you saying you want relationships to custom tables, stored in a separate table than wp_podsrel?

**Comment by jcampbell05** [comment]
Well I think it would be nice if we could :) so its easier to search
relationships manually.
